README.adoc

```diff
@@ -153,34 +153,34 @@ Implement the actual exception class and the mapper class to see how this mechan
 `src/main/java/io/openliberty/guides/inventory/client/UnknownUrlException.java`
 ----
 UnknownUrlException.java
-[source, Java, linenums, role='code_column']
+[source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/client/UnknownUrlException.java[tags=**;!copyright]
+include::finish/src/main/java/io/openliberty/guides/inventory/client/UnknownUrlException.java[]
 ----
 
-Now, link the [hotspot=3-14 file=1]`UnknownUrlException` class with the corresponding response code through a `ResponseExceptionMapper` mapper class.
+Now, link the [hotspot=UnknownUrlException file=1]`UnknownUrlException` class with the corresponding response code through a `ResponseExceptionMapper` mapper class.
 
 [role="code_command hotspot", subs="quotes"]
 ----
 #Create the `UnknownUrlExceptionMapper` class.#
```

Comment:
This block highlights the `SystemClient.java` file instead of the `UnknownUrlException.java` file

Reply:
Opened a separate issue since README looks correct (also an issue on master).